src/FileIndexer.Components.test.ts

```diff
@@ -0,0 +1,251 @@
+import { test } from 'uvu';
+
+import * as lsif from './lsif';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as assert from 'uvu/assert';
+
+import { Input } from './Input';
+import { FileIndexer } from './FileIndexer';
+import * as ts from 'typescript';
+import { ProjectOptions } from './CommandLineOptions';
+import { Counter } from './Counter';
+import { Packages } from './Packages';
+import { LsifSymbol } from './LsifSymbol';
+import { DefinitionRange } from './DefinitionRange';
+import { createLanguageService, ProjectIndexer } from './ProjectIndexer';
+
+let ignoreSkipParameter: boolean = false;
+
+let counter = new Counter();
+let options: ProjectOptions = {
+    cwd: '/Users/ericmeadows/git/echarts',
+    yarnWorkspaces: false,
+    yarnBerryWorkspaces: false,
+    inferTsconfig: false,
+    output: '/Users/ericmeadows/git/lsif-spike/temp/lsif-ts.lsif',
+    progressBar: true,
+    indexedProjects: new Set<string>(['/Users/ericmeadows/git/echarts']),
+    projectRoot: '/Users/ericmeadows/git/echarts',
```

Comment:
Is this going to prevent the tests from running outside of your local environment?

Reply:
It's not even used during execution :)